prettier.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+	arrowParens: 'avoid',
```

Comment:
Looking around our codebase, I see `( x ) => foo( x )` more than I see `x => foo( x )`. Should we set this to `'always'`?

Reply:
Makes sense, I may have [peeked at Calypso](https://github.com/Automattic/calypso-prettier/commit/81c358c081f892db2b2990db2cd21be2fd87cdc2) in the makings of this 😏

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/4628/commits/e258034482749563b10e4dffbe3927c0b6d743d6